src/hash-table.lisp

```diff
@@ -31,16 +31,24 @@
 ;;; strings for each Lisp object. To do this, we tag the objects with
 ;;; a `$$jscl_id' property. As a special case, numbers do not need to
 ;;; be tagged, as they can be used to index Javascript objects.
+
+(/debug "loading hash-table.lisp!")
+
 (defvar *eq-hash-counter* 0)
+
+(defun %concat (&rest elements)
```

Comment:
There is already `concat` defined in `string.lisp`.

Reply:
Yes. I know.
Previously, my hash was determined before the string and other functionality. When the concat itself is not yet defined. Logic dictates that such functions should be moved to the boot. I haven't done it yet. I would like to do everything in a systematic and uniform way. No patching.

Reply:
I thin the order in which the functions are defined should not be important.  Unless they are used in macro definitions!

Reply:
fixed